packages/functional-tests/tests/resetPassword/syncV3ResetPassword.spec.ts

```diff
@@ -2,18 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { expect, test } from '../../../lib/fixtures/standard';
+import { expect, test } from '../../lib/fixtures/standard';
 
 test.describe('severity-1 #smoke', () => {
   test.describe('Firefox Desktop Sync v3 reset password react', () => {
-    test.beforeEach(async ({ pages: { configPage } }) => {
-      const config = await configPage.getConfig();
-      test.skip(
-        config.featureFlags.resetPasswordWithCode !== true,
-        'see FXA-9728, remove conditional skip when feature flag removed'
```

Comment:
These tests have been running for a few months, `resetPasswordWithCode` is set to true in all environments. If we need to skip these tests, we would also need to look at all of the other syncV3 tests that are not skipped for chromium, as far as I can tell?

Reply:
Oh. I thought the `resetPassword/syncV3resetPassword.spec.ts` was being skipped in the past. But if it was always set to true then it would be probably fine.
I found this with the recent nightly run for chromium tests: https://app.circleci.com/pipelines/github/mozilla/fxa/54464/workflows/e26432fd-2d64-4740-a41d-3ee2e2275b02/jobs/530856/parallel-runs/4?filterBy=ALL

<img width="1350" alt="Screenshot 2024-09-24 at 11 55 57 AM" src="https://github.com/user-attachments/assets/1ae55aa8-86b9-4ddb-98a8-0fe932ff6506">

Reply:
ah, you already removed the `resetPasswordWithLink` test so it wouldn't matter. Sorry for the confusion. 😿

Reply:
All good, thank you for the review! I did indeed remove all the skipped `resetPasswordWithLink` tests.

I'm uncertain when tests need to be skipped in Chromium for sync - looks like it's only two tests at the moment, which seems surprising